content/configuration/helm-options.md

```diff
@@ -18,7 +18,7 @@ Controller for the BotKube Slack app which helps you monitor your Kubernetes clu
 
```

Comment:
Do we also update the Maintainers section? Or we can even delete it 🤔 Later we can point to CODEOWNERS file on `botkube` repository.

Reply:
delete 👍